intranet/templates/eighth/admin/schedule_activity.html

```diff
@@ -67,12 +67,7 @@
 {% block css %}
     {{ block.super }}
     {% stylesheet 'eighth.schedule' %}
-{% endblock %}
-
-{% block head %}
     {% if dark_mode_enabled %}
```

Comment:
Please add the `{% block head %}` (with `{{ block.super }}`) back. The `dark` stylesheets need to be in the `head` block so they are added last.